content/doc/developer/plugin-development/incrementals.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: developersection
+title: "Developing plugins and Jenkins core in parallel"
```

Comment:
Not sure how to capture the comment from @jglick in the title, but here is one possible idea:
```suggestion
title: "Developing components in parallel"
```